src/Ecas.php

```diff
@@ -156,6 +141,12 @@ public function requestTicketValidation(
      */
     public function supportAuthentication(array $parameters = []): bool
     {
+        $ticket = $this->extractTicket();
```

Comment:
There is no need to check for URI "ticket" params here because it's done already in the parent call, see: https://github.com/ecphp/cas-lib/blob/master/src/AbstractCas.php#L180

Reply:
Fixed in df3c27c